### PR TITLE
some rename in code_8006e9c0

### PR DESCRIPTION
**Pull request by coco875**
rename function and variables mainly related to object

### CODE REVIEW

File: asm/bss_8001C4D0.s [hunk 1]
```diff
@@ -667,76 +667,25 @@ glabel gItemWindowObjectByPlayerId
...
-
-glabel D_80183EEC
-.skip 60
-
-glabel D_80183F28
-.skip 4
-
-glabel D_80183F2C
-.skip 8
+glabel indexObjectList1
```
Comment (MegaMech) [inline asm/bss_8001C4D0.s:670]: objectListIndex1

(I think I prefer)
objectListIndex
objectListIndex2
etc.
but not against objectListIndex1

File: asm/bss_8001C4D0.s [hunk 2]
```diff
@@ -795,9 +744,9 @@ glabel D_8018C970
 .skip 256
 
 # some hud related list begin
-# D_8018CA70 should be of size 528 (0x84 * 0x4)
+# hudPlayers should be of size 528 (0x84 * 0x4)
 # D_8018CA74 through D_8018CC7D are all fake
-glabel D_8018CA70
+glabel hudPlayers
```
Comment (MegaMech) [inline asm/bss_8001C4D0.s:749]: playerHUD ?

File: include/variables.h [hunk 3]
```diff
@@ -15,7 +15,7 @@ extern s32   gLapCountByPlayerId[8];             // D_80164390
 extern s32   gPlayerIsThrottleActive[];           // D_801653E0
 extern s32   gGPCurrentRaceRankByPlayerId[];      // D_801643B8
 
-extern struct_8018CA70_entry D_8018CA70[];
+extern hud_player hudPlayers[];
```
Comment (MegaMech) [inline include/variables.h:18]: player1HUD ?

I'm not opposed to `hud_` being a prefix so long as its used consistently and the rest of the name follows camelCase:
ex: `hud_myVariable`
`hud_player1`

File: src/code_8006E9C0.c [hunk 4]
```diff
@@ -1303,32 +1303,32 @@ void init_hud_one_player(void) {
     find_unused_obj_index(&D_80183DBC);
     find_unused_obj_index(gItemWindowObjectByPlayerId);
     find_unused_obj_index(gItemWindowObjectByPlayerId + 4);
-    func_80070190();
+    init_index_object_list();
```
Comment (MegaMech) [inline src/code_8006E9C0.c:1306]: init_object_list_index

**Review by MegaMech — CHANGES_REQUESTED** [review]

**Comment by MegaMech** [comment]
I checked to see what the build issue is. It doesn't show the error. So you'll have to test locally.

File: asm/bss_8001C4D0.s (hunk 1 above)
Comment (coco875) [reply asm/bss_8001C4D0.s:670]: by naming it objectListIndex1 this implictly mean there is atleast a 2

**Comment by coco875** [comment]
> I checked to see what the build issue is. It doesn't show the error. So you'll have to test locally.

on my computer it's work

File: asm/bss_8001C4D0.s (hunk 1 above)
Comment (MegaMech) [reply asm/bss_8001C4D0.s:670]: fair

**Review by MegaMech — APPROVED** [review]

**Pull request merged**